lib/vrl/web-playground/public/vrl-highlighter.js

```diff
@@ -0,0 +1,636 @@
+// VRL Language Definition
+export let vrlLanguageDefinition = {
+    defaultToken: "invalid",
+    ignoreCase: true,
+    tokenPostfix: ".vrl",
+
+    brackets: [
+        { open: "{", close: "}", token: "delimiter.curly" },
+        { open: "[", close: "]", token: "delimiter.square" },
+        { open: "(", close: ")", token: "delimiter.parenthesis" },
+    ],
+
+    regEx: /\/(?!\/\/)(?:[^\/\\]|\\.)*\/[igm]*/,
+
+    keywords: [
+        "abort",
+        "as",
+        "break",
+        "continue",
+        "else",
+        "false",
+        "for",
+        "if",
+        "impl",
+        "in",
+        "let",
+        "loop",
+        "null",
+        "return",
+        "self",
+        "std",
+        "then",
+        "this",
+        "true",
+        "type",
+        "until",
+        "use",
+        "while",
+    ],
+
+    // we include these common regular expressions
+    symbols: /[=><!~?&%|+\-*\/\^\.,\:]+/,
+    escapes: /\\(?:[abfnrtv\\"'$]|x[0-9A-Fa-f]{1,4}|u[0-9A-Fa-f]{4}|U[0-9A-Fa-f]{8})/,
+
+    // The main tokenizer for our languages
+    tokenizer: {
+        root: [
+            // function invokes fallible
```

Comment:
It would be great if the highlighter could add extra emphasis on the `!` token in a fallible function call. We don't need to do so in this PR, but worth looking at in the future.